rust/libits-client/src/reception/exchange/mobile_perceived_object.rs

```diff
@@ -118,26 +130,78 @@ fn compute_speed(x_speed: i16, y_speed: i16) -> u16 {
     speed_from_yaw_angle(x_speed, y_speed)
 }
 
-fn compute_heading(y_distance: i32, cpm_heading: u16) -> u16 {
-    match y_distance {
-        y if y < 0 => (cpm_heading + 1800) % 3600,
+fn compute_heading_from_mobile(perceived_object: &PerceivedObject, cpm_heading: u16) -> u16 {
```

Comment:
compute_heading_from_vehicle instead?

Reply:
I can change `mobile` for `vehicle` in the signatures    
I can be relevant as it is originating from some "vehicle_container"